src/NNlib.jl

```diff
@@ -2,23 +2,11 @@ module NNlib
 
 using Requires
 
-export
-    σ,
-    relu,
-    leakyrelu,
-    elu,
-    swish,
-    softmax,
-    selu,
-    softplus,
-    softsign,
-    conv2d,
-    conv2d_grad_w,
-    conv2d_grad_x,
-    pool,
-    pool_grad
+export σ, relu, leakyrelu, elu, swish, selu, softplus, softsign,
+  softmax, conv2d, pool
+
+const libnnlib = Libdl.find_library(is_windows() ? "conv.dll" : "conv.so", [@__DIR__])
```

Comment:
You should use `Libdl.dlext` :)

Reply:
In this case the build script actually does .so even on OS X. That could possibly be cleaner though